libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpAdapter.cs

```diff
@@ -139,7 +139,9 @@ private async Task ConnectWebSocketAsync(IBot bot, HttpRequest httpRequest, Http
             try
             {
                 var socket = await httpRequest.HttpContext.WebSockets.AcceptWebSocketAsync().ConfigureAwait(false);
-                var requestHandler = new StreamingRequestHandler(bot, this, socket, _logger);
+                var appId = (_credentialProvider as ConfigurationCredentialProvider)?.AppId;
+                var appPassword = await _credentialProvider.GetAppPasswordAsync(appId).ConfigureAwait(false);
+                var requestHandler = new StreamingRequestHandler(bot, this, new MicrosoftAppCredentials(appId, appPassword), socket, _logger);
```

Comment:
The adapter always has an app credentials, can we use that? People could be using CertiicateAppCredentials. In general it is a bad idea to instantiate MicrosoftAppCredentials, throws injection patterns